src/test/java/org/mockito/internal/handler/InvocationNotifierHandlerTest.java

```diff
@@ -121,11 +120,9 @@ public void should_report_listener_exception() throws Throwable {
     public void should_delegate_all_MockHandlerInterface_to_the_parameterized_MockHandler() throws Exception {
         notifier.getInvocationContainer();
         notifier.getMockSettings();
-        notifier.setAnswersForStubbing(new ArrayList<Answer<?>>());
 
         verify(mockHandler).getInvocationContainer();
         verify(mockHandler).getMockSettings();
-        verify(mockHandler).setAnswersForStubbing(ArgumentMatchers.<Answer<?>>anyList());
```

Comment:
>This is thus a breaking change? :(

"setAnswersForStubbing" method was internal so this is not a breaking change. I removed this line from the test because we no longer need this method :)

Thank you for feedback!